src/functions-reference/covariance_matrix_distributions.Rmd

```diff
@@ -5,6 +5,7 @@ positive-definite $K \times K$ matrices.
 
 ```{r results='asis', echo=FALSE}
 if (knitr::is_html_output()) {
+cat(' * <a href="inverse-wishart-cholesky-distribution.html">Inverse Wishart Distribution, Cholesky Parameterization</a>\n')
```

Comment:
This link doesn't work in the built documentation.  I'm not sure why.  The link points to:

file:///Users/bcarpenter/github/stan-dev/docs/docs/2_30/functions-reference/inverse-wishart-cholesky-distribution.html

But the actual section is:

file:///Users/bcarpenter/github/stan-dev/docs/docs/2_30/functions-reference/inverse-wishart-distribution-cholesky-parameterization-inverse-wishart-cholesky-distribution.html

And the `(#inverse...)` is rendered as part of the title rather than acting as a hook.  I think that's because you want to use braces rather than parens for the links.